stable/testlink/templates/svc.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: "{{ template "testlink.chart" . }}"
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Name | quote }}
+  annotations:
```

Comment:
Use:
```yaml
  {{- if .Values.service.annotations }}
  annotations: {{ include "testlink.tplValue" ( dict "value" .Values.service.annotations "context" $) | nindent 4 }}
  {{- end }}
```
Then, define the **"testlink.tplValue"** macro (if it doesn't exist already). This way, you can evaluate annotations as a template.

Reply:
done